Vastly Cut Down Extraneous Code

Canyon-Artuz opened the pull request:
Shooting is now completely independent.

DualDuelist committed:
Vastly Cut Down Extraneous Code

Rv-ben commented on Assets/Scripts/Charater Scripts/ShootingEnums.cs:
```diff
@@ -0,0 +1,13 @@
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ShootingEnums : MonoBehaviour
```
This should be separated. In this file there should only be enums, no classes. In this case we do not want to make enums an attribute in a class.

Rv-ben commented on Assets/Scripts/Charater Scripts/Shooting.cs:
```diff
@@ -0,0 +1,77 @@
+        {
+            // Instantiates a pre-chosen bullet at specified ejector facing the same way as the ejector.
+            GameObject bull = Instantiate(bullet, ejector.position, ejector.rotation);
+            bull.GetComponent<Rigidbody>().velocity = transform.forward + new Vector3(0, 0, Random.Range(-bullet.GetComponent<Bullet>().getSpread(), bullet.GetComponent<Bullet>().getSpread())) * bullet.GetComponent<Bullet>().getBulletSpeed();
```
Improve this line's readability, this line is too long. Do something like this: https://stackoverflow.com/questions/1263326/break-long-code-lines-in-visual-studio-2008

Rv-ben commented on Assets/Scripts/Charater Scripts/Shooting.cs:
```diff
@@ -0,0 +1,77 @@
+    // Float that determines when the Player is able to shoot again.  Set during run-time in the Shoot() function.
+    public float timeToShoot;
+    // Checks if the weapon projectiles should grow around the Player like a bubble.
+    // public bool grows;
```
Get rid of commented out code

Rv-ben reviewed (changes requested):
Nice work overall, just touch up a few details.

DualDuelist committed:
Fixed Requests

Removed Commented Out Code
Commented all lines.
Split apart the overly long line of code.
